### PR TITLE
allow_multiple_attempts enforcement launch

### PR DESCRIPTION
**Pull request by bethanyaconnor**
This PR does two things:
- Flips the default value for the DCDO flag `enforce_allow_multiple_attempts`
- Adds UI tests testing `allow_multiple_attempts` being true on contained levels and false on standalone levels (the opposite of prior behavior -- the inverse should already be covered by existing tests). The vast majority of this diff is the setup for those tests.

Sauce Labs runs of new tests:
- `Student can attempt retriable free response contained level multiple times`: https://app.saucelabs.com/tests/f38de102130a4ee28e968c0dff990114
- (free response) `Level without multiple attempts allowed is locked after submit`: https://app.saucelabs.com/tests/43472ee409b44db094e5545737aff1d8
- (multi) `Standalone level without retries locks after answer is submitted`: https://app.saucelabs.com/tests/b04a8c4e35824bbebb0718cdaec78f31
- `Student can retry multiple choice contained level that allows multiple attempts`: https://app.saucelabs.com/tests/cf90309e8c8e457199b8fef82cca4040

### CODE REVIEW

File: dashboard/test/ui/features/teacher_tools/level_types/multiple_choice_contained_levels.feature [hunk 1]
```diff
@@ -67,7 +67,7 @@ Scenario: Unauthorized Teacher on Maze with multiple choice contained level
 
 Scenario: Teacher can reset progress on multiple choice contained level
   Given I sign in as "Teacher_Lillian"
-  And I am on "http://studio.code.org/s/allthethings/lessons/41/levels/2?enableExperiments=instructorPredictLevelReset"
+  And I am on "http://studio.code.org/s/allthethings/lessons/41/levels/2"
```
Comment (bethanyaconnor) [inline dashboard/test/ui/features/teacher_tools/level_types/multiple_choice_contained_levels.feature:70]: Just removed this experiment flag while I was here

**Review by rshipp — COMMENTED** [review]
is the failing test just unrelated flakiness? i can't tell. the rest of this looks good to me. thank you for the tests!

**Review by hannahbergam — APPROVED** [review]
LGTM! And yay for tests:)

**Comment by bethanyaconnor** [comment]
> is the failing test just unrelated flakiness

Sorry, it passed Drone a couple times so I sent out for review without waiting for Drone. The failing test looks to be `Chrome_star_labs_gamelab_libraries`, which I believe would be unrelated. I'll rerun Drone to be sure

**Pull request merged**